modello-maven-plugin/src/main/java/org/codehaus/modello/maven/AbstractModelloGeneratorMojo.java

```diff
@@ -95,6 +99,15 @@ public abstract class AbstractModelloGeneratorMojo extends AbstractMojo {
     @Parameter
     private List<String> packagedVersions = new ArrayList<String>();
 
+    /**
+     * The contents of license header, verbatim. It may be file path and if file exists, will be loaded up. Otherwise,
+     * the contents of this parameter is reused as-is.
+     *
+     * @since 2.3.1
+     */
+    @Parameter
+    private String licenseText;
```

Comment:
I would like to consider two parameters:
 - one with plain text
 - one with file name 

Code will be simple and we don't need a magic to discover what content is

Reply:
Agreed, so mojo have two params.... model config has one that holds string (that is loaded from file or was param). My bigger problem is how header is mutilated by XML writer... newlines are getting removed and so on.

Reply:
Also, if two params... what happens if both are configured?

Reply:
someone wins ... or throw an error

Reply:
I documented: licenseText config "prevails" if both set. That's all.